GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/GameManager.cs

```diff
@@ -8,24 +8,17 @@ public class GameManager : MonoBehaviour
     public GameObject playPageCanvas;
     public GameObject storePageCanvas;
     public GameObject garagePageCanvas;
-    public Text coinsCount;
-    public Car sedan;
-
-    private string _filename = "data.json";
-    private string _dataPath;
-    private GameData _gameData;
 
+    public Text coinsCount;
     // Start is called before the first frame update
     public void Start()
     {
         Debug.Log(SecurityController.registerUserDevice());
-        PlayerPrefs.DeleteAll();
+        Debug.Log(SecurityController.verifyAndSaveUserPurchase("tokenId"));
```

Comment:
delete?

Reply:
Yes, I think this modification was made by Zekai as the game no longer use PlayerPrefs, but saves the data in a json file